deps/rabbitmq_stream/src/rabbit_stream_reader.erl

```diff
@@ -146,6 +146,12 @@
          peer_cert_subject,
          peer_cert_validity]).
 
+-ifdef(TEST).
+-define(STATE_TIMEOUT, 500).
+-else.
+-define(STATE_TIMEOUT, 10_000).
```

Comment:
Should we be more specific about what states this covers?

Reply:
Do you mean renaming the `STATE_TIMEOUT` macro?
We could, yes.
That timeout applies to states `tcp_connected`, `peer_properties_exchanged`, `authenticating`, `tuning`, `tuned`, `close_sent`. It does not apply to state `open`.
I can't come up with a good name, though 🤔
Do you have a suggestion?

Reply:
`CONNECTION_NEGOTIATION_STEP_TIMEOUT`?

Reply:
That's a very good name, thank you 🙂
Renamed it.